team2_pii.py

```diff
@@ -21,26 +26,81 @@ def find_credit_card_number(text):
     if match:
         return True
     return False
+
+def is_leap_year(year):
+    if year % 4 == 0:
+        if year % 100 == 0 and year % 400 != 0:
+            return False
+        return True
+    return False
+
+def find_dob(text):
```

Comment:
You would expect a test case such as this to pass since this is not a valid DOB, but it fails
```
        dob = 'SSN: 001-01-0001'
        self.assertFalse(find_dob(dob))
```